src/elements/content-sidebar/MetadataSidebar.js

```diff
@@ -416,6 +420,8 @@ class MetadataSidebar extends React.PureComponent<Props, State> {
                                 onModification={this.onModification}
                                 onRemove={this.onRemove}
                                 onSave={this.onSave}
+                                selectedTemplateKey={selectedTemplateKey}
+                                templates={templates}
```

Comment:
Does this need to be passed in? It doesn't seem to be used.

Reply:
Good catch!